scripts/amplitude/config.js

```diff
@@ -0,0 +1,7 @@
+import { configFile } from "../utils/configFile.js";
+import { HOST_CONFIG_NAME } from "../utils/constants.js";
+
+export const HAS_ASKED_OPT_IN_NAME = "has-asked-opt-in";
+export const OPT_IN_NAME = "opt-in";
+
+export const AMPLITUDE_API_KEY = configFile.get(HOST_CONFIG_NAME) === "https://app.crowdbotics.com/" ? "production-key-placeholder-text" : "development-key-placeholder-text";
```

Comment:
Question: Where will the key be coming from?

Reply:
I was thinking about this, we will probably need to store the production key in the constants file. I don't  love this, but not sure if we can use environment variables, but we could test.

Reply:
Have been looking into this for a bit so didn't respond yet. We basically have to give the user the API key at some point cause they are going to install our package on their machine.

Trying to get a decent solution for this but ultimately it depends on how far we want to go to hide the api keys.

Just noticing that other well used integrations like Zoom, github, etc passes the responsibility to the user. Ie they require you to give you some sort of key to use their services.

Reply:
I am not super worried about amplitude API keys, but I could be in the wrong. I would be ok to make it the default value as a constant, like the default API url is the production url. As far as tracking data usage